targets/targets.json

```diff
@@ -8420,17 +8419,52 @@
             "FLASH",
             "MPU"
         ],
-        "components_add": ["FLASHIAP"],
+        "components_add": [],
         "detect_code": ["1305"],
         "release_versions": ["5"],
         "device_name": "M2351KIAAEES",
         "bootloader_supported": true,
         "tickless-from-us-ticker": true,
         "forced_reset_timeout": 3,
-        "mbed_rom_start"    : "0x10040000",
-        "mbed_rom_size"     : "0x40000",
-        "mbed_ram_start"    : "0x30008000",
-        "mbed_ram_size"     : "0x10000"
+        "public": false
+    },
+    "NUMAKER_PFM_M2351_NOPSA_NS": {
```

Comment:
This name is too long as it also needs to be in the online database (max 20 characters)

Reply:
Fixed